### ISSUE TITLE
Address Range matching -- more refinements

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#617]
Comment by @bravacheLBL from issue #317, which I have closed.
There are still more things that should be addressed
@bravacheLBL  added a new file in Gdrive /Issue 317 (https://drive.google.com/open?id=0B3fTKpZ9Dx7LUlQyUG9YZkQ2aDQ) with the suffix "FORMAT" where I changed the format of some address line and added a field "Note" where I wrote what I changed.

I then tried to match those buildings with the other database named "CoveredBuildings" (that was imported first).

The format changes that matched correctly are:

Deleting a dash that was separated with spaces : "1654 - 1658" -> "1654 1658"
Uncapitalizing letters : "New York AV NW" -> "new york av nw"
Replacing a dash separated by spaces with a slash : "1654 - 1658" -> "1654 / 1658"
The format changes that didn't match are:

Replacing Dash + Space by Slash : "1654 - 1658" -> "1654/1658"
Shortened second part of number : "1654 - 1658" -> "1654 - 58"
Spelled out ST to street or NW to north-west or av to avenue
Replaced a dash by a space: "1654-1658" -> "1654 1658" (this is different from the "deleting dash" that DID match since the initial address doesn't have spaces around in this case)
The format changes that worked once and failed the other time:

Deleting/Adding spaces around dash : "1654 - 1658" -> "1654-1658" (or vice-versa)
I believe we can keep this issue open.

**Comment by RDmitchell** [comment]
I have set the priority = 3 and not assigned a milestone yet. We will wait and see how other things stack up before deciding when to tackle this.

**Comment by RDmitchell** [comment]
Looking at the matching results when the files that have the prefix "Matched" on them, the matching results are a bit mixed. We should revisit the matching results when we have time, using those same files in
https://drive.google.com/open?id=0B3fTKpZ9Dx7LUlQyUG9YZkQ2aDQ

**Comment by RDmitchell** [comment]
Here is another example of an address that it would be nice if the program would match, ie, a street number that falls within a range of addresses

![image](https://cloud.githubusercontent.com/assets/6314950/12276840/d695e374-b92c-11e5-9e13-e4221280ca70.png)

**Comment by nicholasserra** [comment]
A little confused on the values I need to fix. Can I get a list of addresses as "input" -> "desired output"?

**Comment by nicholasserra** [comment]
@RDmitchell still need some clarification on this one.

**Comment by RDmitchell** [comment]
Right. I might not have time to do that until after the Feb release, ie, next week.

**Comment by nicholasserra** [comment]
No worries
